### DISCUSSION TITLE
Blob URL AR Quicklook with custom panel

### DISCUSSION THREAD

**Discussion by ThorstenBux** [google/model-viewer#2567] [Q&A]
Hi,

I'm wondering if someone knows why this doesn't work:
`ios-url="blob:https://192.168.1.134:3000/a4bcda5f-5f78-4604-a81c-16bfe2560574#callToAction=Buynow&checkoutTitle=Georgia-Dashes&checkoutSubtitle=Abeautifulturtle&price=xx"`

I mean the model shows up but the panel doesn't appear.
If I use a link to a USDZ file directly with the same hash the panel appears. 

Has anyone tried to use a blob / data-url to load Quicklook together with a hash string?

I know this is not really a ModelViewer question but I thought maybe someone knows. Any hints highly apprechiated.

**Comment by elalish** [comment]
We had problems with this too. I read somewhere that Safari doesn't support blobs with query parameters. File a bug on Apple!
